models/fetchCommentsFromId.model.js

```diff
@@ -0,0 +1,14 @@
+const db = require('../db');
+
+exports.fetchCommentsFromId = (id) => {
+    if(!/[\d]/.test(id)){
```

Comment:
This is good - but you'll need to handle non-existent id's too, so if the returned rows from the db.query === 0, you can promise.reject "review not found" or similar.